features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -611,6 +624,7 @@ Feature: Collection
 
   # mod reveals authors on Jan 1st
   When I am logged in as "mod1"
+    And all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.